uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/ContactActionsMenu/Page/Inline/ActionsPart2.tpl

```diff
@@ -35,7 +35,10 @@
             <button class="button__delete btn-warning">Disable User Account</button>
           </div>
           <div class="col-md-6">
-            <button class="button__disable btn-danger">Delete User Account</button>
+            <button class="button__disable btn-danger" 
+              data-delete-user-url="civicrm/contactactionsmenu/deleteuseraccount">
```

Comment:
I am not sure if we really need this, we can fetch the contact id from the url query string(cid) itself.
`http://civihr.local:8914/civicrm/contact/view?cid=204&selectedChild=hrjobcontract#/`.

And also as the deletion url is always same, we dont need to mention that everytime in the HTML, we can specify that in the JS itself.
```
var contactId = getItFromQueryString
var profileDeletetionURL = 'civicrm/contactactionsmenu/deleteuseraccount';
...
 window.location = contactId + url;
`

Reply:
cc @AkA84 ^

Reply:
I agree @debarshi,  as the url will never change, it makes sense to create the user account deletion url in `js` code itself rather keeping in markup. In order to parse the contact(cid) form query strings we can use function similar to here: http://jsfiddle.net/bmnepali/576u60r4/

cc @AkA84 ^

Reply:
@deb1990 @bmnepali honestly i'd like more the backend have full control over the url, instead of hardcoding it in the frontend. I'd rather have, not only the id, but the full path injected by the backend (cc @tunbola )

Reply:
Ok! then i will leave it as it is now. So, the backend dev can add the url as I described in description.